second.ino

```diff
@@ -60,15 +60,15 @@ float dt;
 float milliOld_mpu;
 float milliNew_mpu;
 float dt_mpu;
-int error = 0;
+int error = 0; // ?? Why there are so many global variables here while we can use them in functions only 
```

Comment:
You are correct about this, the code is a bit of a mess but it will, one day, be rewritten with some usage of data structures.
We don't mind that you change the code as long as it doesn't make some compilation errors or warnings, it's only for review for *tomorrow*